Sources/RxAlamofire.swift

```diff
@@ -743,6 +743,58 @@ extension ObservableType where E == DataRequest {
     public func responseJSON() -> Observable<DataResponse<Any>> {
         return self.flatMap { $0.rx.responseJSON() }
     }
+    
+    public func json(options: JSONSerialization.ReadingOptions = .allowFragments) -> Observable<Any> {
+        return self.flatMap { $0.rx.json(options: options) }
```

Comment:
I'm not sure if all of the `self.` were in the original code but they're not really needed in this context 🤔 
Anyways it's fine, I think this isn't on you but was just a think done in previous implementations.

Reply:
I was inspired by previous code but it can be removed yep.